git-submodules-status

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
+
+################################################################################
+#
+#   Description:
+#   ------------
+#       Fetches all submodules from their respective remote repositories, and
+#       shows a table that lists for each submodule : the latest available
+#       version, the current version and a small visual cue to easily identify
+#       mismatches between the two.
+#
+#   Note that this script assumes that a project's submodules are present in a
+#   directory called "submodules" in the project's top-level git directory.
```

Comment:
I don't think this is true anymore, it seems like the information about the submodules is being retrieved from `.gitmodules`.